congrats.json

```diff
@@ -8,4 +8,5 @@
   "You shipped it 🚀, way to go on your first package!!!",
   "👏🏿 congrats 👏🏻 on 👏🏾 your 👏🏼 first 👏🏽 package 👏",
   "It's a good package, Bront 🐶👍💯"
+  "Welcome to the npm family, bro"
```

Comment:
hey! thanks for contributing! not everyone who publishes will identify as "bro" tho- do you think you could update it to something gender neutral? thanks!